src/interface/src/app/treatments/direct-impacts/direct-impacts.component.html

```diff
@@ -46,15 +48,25 @@ <h4 class="title">Direct Treatment Impacts</h4>
 
       <sg-panel
         class="map-panel"
+        [ngClass]="{ prescriptions: showTreatmentPrescription }"
         [paddedContent]="false"
         [buttons]="[
           { icon: 'layers', actionName: 'layers' },
```

Comment:
minor thing I missed before: should this layer icon be in outline style?

Reply:
great catch! it should, thanks! updating